go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	github.com/hashicorp/go-syslog v1.0.0
 	github.com/hashicorp/go-uuid v1.0.2
 	github.com/hashicorp/go-version v1.3.0
-	github.com/hashicorp/hcat v0.2.1-0.20220105214302-4b15127c5629
+	github.com/hashicorp/hcat v0.2.1-0.20220112200956-444bf3805a7a
```

Comment:
This is currently pointed to this branch https://github.com/hashicorp/hcat/pull/84 that was recently rebased

Reply:
Updated to master + updated Watcher interface https://github.com/hashicorp/consul-terraform-sync/pull/535/commits/8b18d90e96e875e3041fae26b8131b356d849454